src/sentry/event_manager.py

```diff
@@ -826,6 +828,24 @@ def _get_or_create_release_many(jobs, projects):
                     pop_tag(job["data"], "dist")
                     set_tag(job["data"], "sentry:dist", job["dist"].name)
 
+                # Dynamic Sampling - Boosting latest release functionality
+                if (
+                    options.get("dynamic-sampling:boost-latest-release")
+                    and data.get("type") == "transaction"
+                ):
+                    with sentry_sdk.start_span(
+                        op="event_manager.dynamic_sampling_observe_latest_release"
+                    ), metrics.timer("event_manager.dynamic_sampling_observe_latest_release"):
+                        try:
+                            release_observed_in_last_24h = observe_release(project_id, release.id)
```

Comment:
i know it's less important, but if we change `ONE_DAY_TIMEOUT` in `src/sentry/dynamic_sampling/latest_release_booster.py` this variable name will be not accurate. 

We can polish it later.